doc/API/centreon-api-v2.1.yaml

```diff
@@ -0,0 +1,4656 @@
+openapi: 3.0.1
```

Comment:
should we add a documentation `2.1` before the story of changing the versionning strategy ?